src/test/py/bazel/bazel_windows_cpp_test.py

```diff
@@ -592,6 +592,56 @@ def testCppErrorShouldBeVisible(self):
     self.AssertExitCode(exit_code, 1, stderr)
     self.assertIn('this_is_an_error', ''.join(stdout))
 
+  def testBuildWithClangClByCompilerFlag(self):
```

Comment:
Could you either disable these tests until the CI machines have Clang, or wait with merging this PR until then?

If you merge now, everyone's presubmits will fail on Windows because of these tests.

Reply:
Yes, I'll wait until the CI machines have Clang.